pkg/gce-pd-csi-driver/utils.go

```diff
@@ -49,12 +50,12 @@ func NewNodeServiceCapability(cap csi.NodeServiceCapability_RPC_Type) *csi.NodeS
 
 func logGRPC(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
```

Comment:
i think there's actually a logGRPC function under csi-lib-utils/rpc if you want

Reply:
it is the client side version. since pd driver is server side we have to use a different logGRPC